bessctl/cli.py

```diff
@@ -441,6 +430,52 @@ def restore_echoctl(self):
         except:
             pass
 
+    def go_interactive(self):
```

Comment:
I decided to keep "go_interactive" because "maybe_go_interactive" suggests that I can call this function when not in interactive mode and come out still not in interactive mode. I did add a check that if I am already in interactive mode, I should return and not, e.g., re-load the history.